examples/erc20-flash-mint/src/ERC3156FlashBorrowerMock.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "../../solidity-contracts/contracts/interfaces/IERC3156FlashBorrower.sol";
```

Comment:
Can't we import them from by url like [here](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/987d3a3496bcdce46fa11a27c2ec54dbec113a0c/examples/erc721/src/ERC721ReceiverMock.sol#L5)

Reply:
`solc` doesn't work with urls. The reason we can do it in other Solidity files is that those files are, to be frank, useless. 
I'm not even sure why we have them in the repo tbh :thinking:

Reply:
Removed the file as [benches are unnecessary](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/407#discussion_r1898436465)